UM/FileProvider.py

```diff
@@ -39,4 +39,8 @@ def __init__(self) -> None:
 
     def run(self) -> None:
         """Call function associated with the file provider"""
-        raise NotImplementedError
\ No newline at end of file
+        raise NotImplementedError
+
+    def getPriority(self) -> int:
```

Comment:
We can make the `priority` a property of the FileProvider to be consistent with the other properties.